beetle/helpers/sleep.js

```diff
@@ -0,0 +1,11 @@
+function sleep(milliseconds) {
```

Comment:
Couldn't we use setTimeout here? Having 1e7 hardcoded doesn't seem right..

Reply:
Just deleted the helper as it is no longer needed. # @